scripts/github-cache-cli/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vcpkg-github-cache-cli",
+  "version": "1.0.0",
+  "description": "GitHub Actions cache CLI tool for vcpkg binary caching",
+  "main": "github-cache-cli.js",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "node github-cache-cli.js check",
+    "clean": "rm -f *.js *.d.ts *.js.map"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vcpkg-tool.git"
+  },
+  "keywords": [
+    "vcpkg",
+    "github-actions",
+    "cache",
+    "binary-cache"
+  ],
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/cache": "^3.2.4"
```

Comment:
This may be why your node_modules is like half the size of mine

<img width="473" height="848" alt="Image" src="https://github.com/user-attachments/assets/b0d290f3-98c2-4ef0-b321-8829b34c523f" />